test/Table_Tests/src/In_Memory/Split_Tokenize_Spec.enso

```diff
@@ -115,6 +115,18 @@ spec =
             t2 = t.tokenize_to_rows "bar" "[bc]" case_sensitivity=Case_Sensitivity.Insensitive
             t2.should_equal expected
 
+        Test.specify "can do tokenize_to_rows with at_least_one=True" <|
+            input = Table.from_rows ["foo", "bar"] [[0, "a12b34r5"], [1, "qqq"], [2, "2r4r55"]]
+            expected = Table.from_rows ["foo", "bar"] [[0, "12"], [0, "34"], [0, "5"], [1, Nothing], [2, "2"], [2, "4"], [2, "55"]]
+            actual = input.tokenize_to_rows "bar" "\d+" at_least_one=True
+            actual.should_equal expected
```

Comment:
Can we have one test where instead of no matches due to Regex not matching, we just have no matches because the entry was `Nothing` instead of some string?

i.e.
```
        Test.specify "can do tokenize_to_rows with at_least_one=True" <|
            input = Table.from_rows ["foo", "bar"] [[0, "a12b34r5"], [1, Nothing], [2, "2r4r55"]]
            expected = Table.from_rows ["foo", "bar"] [[0, "12"], [0, "34"], [0, "5"], [1, Nothing], [2, "2"], [2, "4"], [2, "55"]]
            actual = input.tokenize_to_rows "bar" "\d+" at_least_one=True
            actual.should_equal expected
```

Reply:
I added this to the same test as another input row.